src/layers/src/geojson-layer/geojson-utils.ts

```diff
@@ -138,6 +141,17 @@ export function parseGeometryFromString(geoString: string): Feature | null {
     }
   }
 
+  // try parse as wkb using loaders.gl WKBLoader
+  if (!parsedGeo) {
+    try {
+      const buffer = Buffer.from(geoString, 'hex');
+      const binaryGeo = parseSync(buffer, WKBLoader);
```

Comment:
This looks great. We should probably just use the `WKTLoader` from loaders.gl/wkt for the wkt parsing on line 138. It is based on wellknown so it should be equivalent but keeping consistent API.

Reply:
Agreed. Will try to use WKTLoader instead of wktParser from wellknown package. Thanks!